app/controllers/creas_strategist_controller.rb

```diff
@@ -15,34 +15,66 @@ def create
     if result.success?
       respond_to do |format|
         format.html { redirect_to planning_path(plan_id: result.plan.id), status: :see_other }
-        format.json { render json: { success: true, plan: result.plan, redirect_url: planning_path(plan_id: result.plan.id) } }
+        format.json {
+          render json: {
+            success: true,
+            plan: serialize_plan(result.plan),
+            redirect_url: planning_path(plan_id: result.plan.id)
+          }
+        }
       end
     else
       respond_to do |format|
         format.html { redirect_to planning_path, alert: result.error }
-        format.json { render json: { success: false, error: result.error }, status: :unprocessable_content }
+        format.json { render json: { success: false, error: result.error }, status: :unprocessable_entity }
       end
     end
   end
 
   private
 
   def find_brand
+    Rails.logger.info "=== DEBUG find_brand ==="
```

Comment:
if this is really for debugging purposes, use Rails.logger.debug instead, thus when running in production the logs won't be polluted by these messages.